### PR TITLE
Replaces deprecated build image

### PR DESCRIPTION
**Pull request by RussKie**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6526)

### CODE REVIEW

**Review by mmitche — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -126,7 +126,7 @@ stages:
       parameters:
         publishUsingPipelines: true
         pool:
-          vmImage: vs2017-win2016
+          vmImage: windows-2019
```
Comment (mmitche) [inline azure-pipelines.yml:129]: Actually, this needs to be replaced with the 1ES pool like the email indicated.

Comment (mmitche) [reply azure-pipelines.yml:129]: name: NetCore1ESPool-Internal
demands: ImageOverride -equals Build.Server.Amd64.VS2019

**Review by mmitche — CHANGES_REQUESTED** [review]
We should use the 1ES pools here.

File: azure-pipelines.yml (hunk 1 above)
Comment (RussKie) [reply azure-pipelines.yml]: Updated

**Review by mmitche — APPROVED** [review]

**Pull request merged**